docs/dashticzconfiguration.rst

```diff
@@ -232,6 +232,9 @@ Config parameters
   * - hide_off_button
     - | 0 / 1
       | Hide off button of selector switch
+  * - speak_lang
+    - | Text to speech language
+      | ``'<LANGUAGE>'``
```

Comment:
Can you add an example how to set the language? Is that 'Dutch', 'NL','NL-nl', something else?

Reply:
These are all the options in the settings.js: de-DE, en-US, es-ES, fr-FR, it-IT, nl-NL, pl-PL, ru-RU